alternatingCase/alternatingCase.js

```diff
@@ -0,0 +1,16 @@
+require("../toUpper/toUpper");
+require("../toLower/toLower");
+
+//AlternatingCase returns the letters in alternating cases starting with a lowercase
+String.prototype.alternatingCase = function() {
+  var alternatingCase = "";//final output initialized to an empty string
```

Comment:
This style of commenting is against AirBnB style. Comments should be above the code it's intended for. Make this correction across the code